.github/main.workflow

```diff
@@ -20,3 +20,14 @@ action "pr-label" {
         LABEL_SPEC_FILE = ".github/PULL_REQUEST_LABELS.yml"
     }
 }
+
+workflow "Issue triage" {
+  on = "issue"
```

Comment:
I believe the (confusing) correct value for this argument is `issues` according to the [GitHub Actions documentation](https://developer.github.com/actions/managing-workflows/workflow-configuration-options/#events-supported-in-workflow-files):

```suggestion
  on = "issues"
```

It appears the documentation in the https://github.com/actions/github repository has the same issue 😅

Reply:
Ha! Thanks for catching that.